core/src/test/resources/ExplicitImplicit/basic.source

```diff
@@ -44,14 +71,124 @@ class A {
 class A {
   class B { class C }
   implicit val x = new B
-  implicit val y = new x.C
+  implicit val y: x.C = new x.C
 }
 >>>
 class A {
   class B { class C }
   implicit val x: B = new B
   implicit val y: x.C = new x.C
 }
+<<< NOWRAP higher kinded cats
+package cats {
```

Comment:
No worries, let's keep it here. It's just such a big input, would be nice to have it smaller.